<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabSetup">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxScenario">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>431</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelScenario">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scenario</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStart_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSimulate">
     <attribute name="title">
      <string>Simulate</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>151</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Infusion Pump </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMedicine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelMedicine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Medicine</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAdjust">
     <attribute name="title">
      <string>Adjust</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsViewWave">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>541</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSliderRate">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>46</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>541</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ECG Rate</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStart_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>infusionPump(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderRate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderRate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
  <slot>stop()</slot>
  <slot>infusionPump(bool)</slot>
 </slots>
</ui>
